docs/index.rst

```diff
@@ -1,20 +1,18 @@
 :mod:`pyramid_nacl_session`
 ===========================
 
-This Pyramid add-on defines an encrypting, pickle-based cookie serializer,
-using ``PyNaCl`` [1] to generate the symmetric encryption for the cookie state.
-
+This Pyramid add-on defines an encrypting, pickle-based cookie serializer, using ``PyNaCl`` [1] to generate the symmetric encryption for the cookie state.
```

Comment:
Please wrap at 80 characters.

Reply:
ah, I thought documentation should not adhere to PEP8 maximum line length (https://docs.pylonsproject.org/projects/docs-style-guide/#use-of-whitespace)